<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpringEditor</class>
 <widget class="QWidget" name="SpringEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>218</width>
    <height>192</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SpringEditor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true">color:rgb(255, 255, 0)</string>
          </property>
          <property name="text">
           <string>Section Radius</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="doubleSpinBoxSecRad">
          <property name="styleSheet">
           <string notr="true">color:rgb(255,255,0);
background-color: rgba(255, 255, 255, 10%);</string>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="minimum">
           <double>0.001000000000000</double>
          </property>
          <property name="maximum">
           <double>99999.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>10.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="styleSheet">
           <string notr="true">color:rgb(255, 255, 0)</string>
          </property>
          <property name="text">
           <string>Coil Radius</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="doubleSpinBoxCoilRad">
          <property name="styleSheet">
           <string notr="true">color:rgb(255,255,0);
background-color: rgba(255, 255, 255, 10%);</string>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="minimum">
           <double>0.001000000000000</double>
          </property>
          <property name="maximum">
           <double>99999.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>30.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="styleSheet">
           <string notr="true">color:rgb(255, 255, 0)</string>
          </property>
          <property name="text">
           <string>Pitch</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="doubleSpinBoxPitch">
          <property name="styleSheet">
           <string notr="true">color:rgb(255,255,0);
background-color: rgba(255, 255, 255, 10%);</string>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="minimum">
           <double>0.001000000000000</double>
          </property>
          <property name="maximum">
           <double>99999.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>20.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="styleSheet">
           <string notr="true">color:rgb(255, 255, 0)</string>
          </property>
          <property name="text">
           <string>Turns</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QDoubleSpinBox" name="doubleSpinBoxTurns">
          <property name="styleSheet">
           <string notr="true">color:rgb(255,255,0);
background-color: rgba(255, 255, 255, 10%);</string>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="minimum">
           <double>0.001000000000000</double>
          </property>
          <property name="maximum">
           <double>99999.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>5.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
